acceptance/acceptance_test.go

```diff
@@ -498,6 +498,52 @@ func testAcceptance(t *testing.T, when spec.G, it spec.S, packFixturesDir, packP
 						})
 					})
 
+					when("--volume", func() {
+						var buildpackTgz, tempVolume string
+
+						it.Before(func() {
+							buildpackTgz = h.CreateTGZ(t, filepath.Join(bpDir, "volume-buildpack"), "./", 0755)
+
+							var err error
+							tempVolume, err = ioutil.TempDir("", "my-volume-mount-source")
+							h.AssertNil(t, err)
+							h.AssertNil(t, os.Chmod(tempVolume, 0755)) // Override umask
+
+							// Some OSes (like macOS) use symlinks for the standard temp dir.
+							// Resolve it so it can be properly mounted by the Docker daemon.
+							tempVolume, err = filepath.EvalSymlinks(tempVolume)
```

Comment:
Are we sure this is necessary since we found the issue caused by umask? If so, would this mean that a standard using on macOS wouldn't be able to mount a `/tmp/dir` or other path with a symlink?

Reply:
@jromero This is a separate issue. It has to do with the fact hat on macOS (at least), the temp dir generated is actually a symlink to something that's not typically configured in Docker preferences unless you add it manually.

Reply:
It seems better to not require folks to update their Docker prefs just to run these tests (and in CI we'd have to figure out how to adjust them too). It would be expected that if they want to mount a temp dir for their own mounting that they'd either use the unsymlinked path, or configure Docker.